controllers/util/storageconsumer.go

```diff
@@ -85,6 +87,12 @@ type StorageConsumerResources interface {
 	ReplaceNfsClientProfileName(string)
 }
 
+type CommonClassEntries interface {
+	GetName() string
+	GetRename() string
+	GetAliases() []string
+}
```

Comment:
Why is this defined here and not in the types file where CommonClassSpec is defined?

Reply:
It made more sense to me that the util wanted the functionality but it's not the work of api provider to club this into a interface rather provide only the blocks we can move into a interface.

Reply:
Ok, but please change the name to something more fitting
```suggestion
type CommonClassSpecAccessors interface {
	GetName() string
	GetRename() string
	GetAliases() []string
}
```